src/ttt/board.clj

```diff
@@ -0,0 +1,31 @@
+(ns ttt.board
+  (:require [clojure.math.numeric-tower :as math]))
+
+(defn initial-board
+  ([board-size]
+   (into [] (range (* board-size board-size))))
+  ([]
+   (initial-board 3)))
+
+
+(defn place-marker
+  [board position marker]
+  (assoc board position marker))
+
+(defn position_available?
+  [board position]
+  (= (nth board position) position))
+
+(defn valid?
+  [position]
+  (boolean (some #(= position %) (initial-board))))
```

Comment:
would there be an issue if you did this with the current board instead of the initial board?

Reply:
Oh if I use the current board, then it will also guard against non-numerical characters and i can delete the other function!